medusa/indexers/tmdb/tmdb.py

```diff
@@ -270,9 +270,9 @@ def _get_episodes(self, tmdb_id, specials=False, aired_season=None):  # pylint:
             else:
                 seasnum, epno = cur_ep.get('seasonnumber'), cur_ep.get('episodenumber')
                 if self.config['dvdorder']:
-                    logger.warning('Episode doest not have DVD ordering available (season: %s, episode: %s). '
+                    logger.warning("Episode doesn't have DVD order available (season: %s, episode: %s). "
```

Comment:
its an alternative yes. but its hard to find the log message while debugging, so i prefer do this